recipes/aws-c-sdkutils/all/conanfile.py

```diff
@@ -1,18 +1,17 @@
-from conans import ConanFile, CMake, tools
+from conan import ConanFile
+from conan.tools.files import get, copy, rmdir
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
 import os
 
-required_conan_version = ">=1.43.0"
-
+required_conan_version = ">=1.52.0"
 
 class AwsCSDKUtils(ConanFile):
     name = "aws-c-sdkutils"
-    description = "aws c language sdk utility library."
-    topics = ("aws", "amazon", "cloud", "utility")
+    description = "C99 library implementing AWS SDK specific utilities. Includes utilities for ARN parsing, reading AWS profiles, etc..."
```

Comment:
```suggestion
description = "C99 library implementing AWS SDK specific utilities. Includes utilities for ARN parsing, reading AWS profiles, etc."
```

I think you can remove the ellipsis.